### DISCUSSION TITLE
Color Pallet Change

### DISCUSSION THREAD

**Discussion by Duphasdan** [lllyasviel/ControlNet#491] [Ideas]
The idea behind this is to simply change the color pallet of of a selected picture to one placed in the Control Net area.  Depending on options, it can change the colors of the whole image, change it depending what area is masked with inpaint, colors changed using the deph leres option, among other options. 

It would take something like, for example, a picture of Superman and change just the coloring to match the coloring used in a picture of Goku from Dragonball Z.  Like, changing the blue of the Superman suit to be orange, black, and white instead of the blue, red, and yellow. Also, the skin color would change to match Goku's skin color.  It would perhaps even detect what the eye colors are and change that as well.

**Comment by geroldmeisinger** [comment]
https://huggingface.co/thibaud/controlnet-sd21 -> ControlNet Color
